src/popup/router/pages/Transactions.vue

```diff
@@ -84,6 +84,7 @@ export default {
       window.removeEventListener('scroll', checkLoadMore);
       clearInterval(polling);
     });
+    this.$watch(({ transactions, displayMode }) => [transactions, displayMode], checkLoadMore);
```

Comment:
```suggestion
    this.$watch(({ displayMode }) => displayMode, checkLoadMore);
```
isn't it enough to subscribe to `displayMode` only?